src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4020,6 +4027,133 @@ def _load_stacks_hardcoded(self):
         self._stacks = stacks
 
 
+class _FlexFunctionAppStackRuntimeHelper:
+    class Runtime:
+        def __init__(self, name, version, app_insights=False, default=False, sku=None, 
+                     end_of_life_date=None, runtime_info=None, github_actions_properties=None):
+            self.name = name
+            self.version = version
+            self.app_insights = app_insights
+            self.default = default
+            self.sku = sku
+            self.end_of_life_date = end_of_life_date
+            self.runtime_info = runtime_info
```

Comment:
I don't think we need `runtime_info` either. All that information can be found in `sku` and you are not even using this anywhere.

Reply:
Yeah! I wanted to use it instead of SKU but it made more sense for Flex version, but I ended up using SKU for the consistency. It will take it out.